crypto/keys/secp256r1/init.go

```diff
@@ -0,0 +1,35 @@
+// secp256r1 package implements Cosmos-SDK compatible ECDSA public and private key. The keys
+// can be protobuf serialized and packed in Any.
+package secp256r1
+
+import (
+	"crypto/elliptic"
+	"fmt"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+)
+
+const (
+	// fieldSize is the curve domain size.
+	fieldSize  = 32
+	pubKeySize = fieldSize + 1
+
+	name = "secp256r1"
+)
+
+var secp256r1 elliptic.Curve
+
+func init() {
+	secp256r1 = elliptic.P256()
+	// pubKeySize is ceil of field bit size + 1 for the sign
+	expected := (secp256r1.Params().BitSize + 7) / 8
+	if expected != fieldSize {
+		panic(fmt.Sprintf("Wrong fieldSize=%d, expecting=%d", fieldSize, expected))
```

Comment:
```suggestion
		panic(fmt.Sprintf("Wrong secp256r1 curve fieldSize=%d, expecting=%d", fieldSize, expected))
```

Reply:
I don't know if this can happen, but if I see this error I would not know what happened

Reply:
This is to cache the curve parameter to shave few microseconds ;)